dashboard/index.html

```diff
@@ -6,149 +6,46 @@
       body {
         color: #333;
         font-size: 12px;
-        padding: 5px;
-      }
-
-      .bars-heading {
-        margin: 0 0 5px 0;
+        padding: 0;
       }
-      .bars-container {
-        border-bottom: 2px #d3d3d3 solid;
-        margin-top: 20px;
+      .charts {
         padding: 10px;
       }
-      .bars {
```

Comment:
There is no css, you [do it through JS](https://github.com/guardian/video-dashboard/pull/8/files#diff-a5f322c1a74739662c914c365c78f6beR17), as it's all SVG etc, although I suppose you could style the it with CSS.